iroh-net/src/magicsock.rs

```diff
@@ -51,25 +45,22 @@ use tracing::{
 use url::Url;
 use watchable::Watchable;
 
-use crate::{
-    defaults::timeouts::NETCHECK_REPORT_TIMEOUT,
-    disco::{self, CallMeMaybe, SendAddr},
-    discovery::{Discovery, DiscoveryItem},
-    dns::DnsResolver,
-    endpoint::NodeAddr,
-    key::{PublicKey, SecretKey, SharedSecret},
-    net::{interfaces, ip::LocalAddresses, netmon},
-    netcheck, portmapper,
-    relay::{RelayMap, RelayUrl},
-    stun, AddrInfo,
-};
-
-use self::{
-    metrics::Metrics as MagicsockMetrics,
-    node_map::{NodeMap, PingAction, PingRole, SendPing},
-    relay_actor::{RelayActor, RelayActorMessage, RelayReadResult},
-    udp_conn::UdpConn,
-};
+use crate::defaults::timeouts::NETCHECK_REPORT_TIMEOUT;
+use crate::disco::{self, CallMeMaybe, SendAddr};
+use crate::discovery::{Discovery, DiscoveryItem};
+use crate::dns::DnsResolver;
+use crate::endpoint::NodeAddr;
+use crate::key::{PublicKey, SecretKey, SharedSecret};
+use crate::net::ip::LocalAddresses;
+use crate::net::{interfaces, netmon};
+use crate::relay::{RelayMap, RelayUrl};
+use crate::{netcheck, portmapper};
+use crate::{stun, AddrInfo};
```

Comment:
uhh do we prefer the import granularity of `module`? I'm sure `crate` is far more common

Reply:
moved to https://github.com/n0-computer/iroh/pull/2812

Reply:
formatter now takes care of this :tada: